packages/components/date-picker/src/date-picker-com/panel-date-pick.vue

```diff
@@ -182,6 +182,7 @@ import {
   DArrowRight,
   ArrowRight,
 } from '@element-plus/icons-vue'
+import { TOOLTIP_INJECTION_KEY } from '@element-plus/components/tooltip/src/tokens'
```

Comment:
I thin we can skip the `src/tokens` part, with only `@element-plus/components/tooltip` should be fine.